javascript-light-bulb/main.js

```diff
@@ -0,0 +1,13 @@
+var $button = document.querySelector('.on-button');
+var $background = document.querySelector('.background-light');
+function lights(event) {
+  if ($button.className === 'on-button') {
```

Comment:
Rather than making this functionality dependent on the current state of the output, you should have a variable here in the JavaScript to tell you if the light is currently on or off. What if the className got changed to an unexpected value by a future addition to the code? Everything would break 💀